File: api/HomeBudget.API/HomeBudget.Application/DTOs/PessoaDtos/CreatePessoaDto.cs
namespace HomeBudget.Application.DTOs.PessoaDtos
{
    public class CreatePessoaDto
    {
        public Guid Id { get; set; }
        public required string Nome { get; set; }
        public required string Cpf { get; set; }
        public int Idade { get; set; }

    }
}

File: web/src/interfaces/PessoaDtos/CreatePessoaDto.ts
import { empty_Guid } from "../../utils/guid";

export class CreatePessoaDto {
  id: string = empty_Guid;
  nome: string = "";
  idade: number = 0;
  cpf: string = "";

  constructor(
    nome: string = "",
    cpf: string = "",
    idade: number = 0,
  ) {
    this.nome = nome;
    this.idade = idade;
    this.cpf = cpf;
  }
}
